deploy/scripts/default.yml

```diff
@@ -1,19 +1,19 @@
 version: '3'
 services:
   opcserver0:
-    image: mcr.microsoft.com/iotedge/opc-plc:latest
+    image: mcr.microsoft.com/iotedge/opc-plc:2.2.0
```

Comment:
Why not use `latest` here ?

Reply:
Latest is broken because it (correctly) validates its own certificate against it's host name, which is always wrong in how we deploy.  The fix (future) is to add the IP to the certificate somehow.  However, it is a larger work item.